src/NuGet.Clients/NuGet.PackageManagement.UI/Options/ConfigurationFilesControl.xaml.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System.Threading;
+using System.Windows.Controls;
+using System.Windows.Documents;
+using System.Windows.Input;
+
+namespace NuGet.PackageManagement.UI.Options
+{
+    /// <summary>
+    /// Interaction logic for ConfigurationFilesControl.xaml
+    /// </summary>
+    public partial class ConfigurationFilesControl : UserControl
+    {
+        private ConfigurationFilesViewModel _viewModel;
+
+        public ConfigurationFilesControl(ConfigurationFilesViewModel configurationFiles)
+        {
+            InitializeComponent();
+            _viewModel = configurationFiles;
+            DataContext = configurationFiles;
+        }
+
+        private void ListViewItem_MouseDoubleClick(object sender, MouseButtonEventArgs e)
+        {
+            _openButton.Command.Execute(null);
+        }
+
+        internal void InitializeOnActivated(CancellationToken cancellationToken)
+        {
+            _viewModel.SetConfigPaths();
+        }
+
+        private void ExecuteOpenExternalLink(object sender, ExecutedRoutedEventArgs e)
+        {
+            if (e.OriginalSource is Hyperlink hyperlink && hyperlink.NavigateUri != null)
```

Comment:
Currently this method only handles when `e.OriginalSource` is of type `Hyperlink`. If this method gets called with any other type, it will silently do nothing. Therefore, if someone tries to use it in a new control, without testing, from users point of view they click something and nothing happens, and our telemetry won't show anything either.

As far as I can tell, this method is only called when a specific `RoutedCommand` is executed. Therefore, it shouldn't be executed when other links or buttons are clicked. Therefore, if we make this command handler throw an exception, or log fault telemetry, I don't expect it will have unintended consequences.